src/CommonLib/CommonLib.vcxproj

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Common lib is for any shared components between AspNetCore and the RequestHandler projects.